ee_tests/src/page_objects/user_profile.page.ts

```diff
@@ -59,26 +60,63 @@ export class CleanupUserEnvPage extends AppPage {
     let confirmationBox = new CleanupConfirmationModal(confirmationElement);
 
     await confirmationBox.ready();
+
     await confirmationBox.confirmationInput.enterText(username);
     await confirmationBox.confirmEraseButton.clickWhenReady();
-    await browser.wait(until.presenceOf(
+
+    await browser.wait(until.presenceOf(this.alertBox), support.LONG_WAIT);
+    let alertText = await this.alertBox.getText();
+    support.info('Alert text = ' + alertText);
+
+    /* An intermittent error is resulting in some user enviroment resets failing
+       https://github.com/openshiftio/openshift.io/issues/1637
+       If this happens - try one more time - if that fails something serious
+       has happened and the test will/and should fail */
+
+    if (alertText.includes('Your OpenShift.io environment has been erased')) {
+      await browser.wait(until.presenceOf(
+      element(by.cssContainingText('fabric8-cleanup', 'Your OpenShift.io environment has been erased!'))));
+      support.info('***Env reset SUCCESS');
+    } else {
+      support.info('***Env reset FAILURE');
+
+      /* Retry - once - to reset env */
+      await this.dashboardButton.clickWhenReady();
+      let dashboardPage = new MainDashboardPage();
+      let userProfilePage = await dashboardPage.gotoUserProfile();
+      let editProfilePage = await userProfilePage.gotoEditProfile();
+      let cleanupEnvPage = await editProfilePage.gotoResetEnvironment();
+
+      await this.eraseEnvButton.clickWhenReady();
+      let confirmationElement1 =  this.innerElement(ui.BaseElement, 'modal', '');
+      let confirmationBox1 = new CleanupConfirmationModal(confirmationElement1);
+
+      await confirmationBox1.ready();
+      await confirmationBox1.confirmationInput.enterText(username);
+      await confirmationBox1.confirmEraseButton.clickWhenReady();
+
+      /* Check for success - if this check also fails - the test will fail */
+      await browser.wait(until.presenceOf(
         element(by.cssContainingText('fabric8-cleanup', 'Your OpenShift.io environment has been erased!'))));
+    }
+  }
 
-    /* Accessing the alert box is failing randomly. Commenting out this code for the
-       short term as the entire reset env feature may be removed in the near future */
-//    support.debug('... waiting for alert box');
-//    await this.alertBox.untilPresent(15 * 1000);
-//    support.debug('... waiting for alert box - OK');
-//    await this.alertBox.untilTextIsPresent('Your OpenShift.io environment has been erased!');
+  async cleanupSupport (username: string) {
+    await this.eraseEnvButton.clickWhenReady();
+    let confirmationElement =  this.innerElement(ui.BaseElement, 'modal', '');
+    let confirmationBox = new CleanupConfirmationModal(confirmationElement);
+    await confirmationBox.ready();
+    await confirmationBox.confirmationInput.enterText(username);
+    await confirmationBox.confirmEraseButton.clickWhenReady();
   }
 
 }
 
+
```

Comment:
This line is not necessary

Reply:
Agreed - that was a typo - removed. Thx!